src/rlx_prv_relup.erl

```diff
@@ -145,7 +155,14 @@ make_upfrom_script(State, Release, UpFrom) ->
                             "relup successfully created!"),
             {ok, State};
         {ok,_, Module,Warnings} ->
-            ?RLX_ERROR({relup_script_generation_warn, Module, Warnings});
+            case WarningsAsErrors of
```

Comment:
Ok, thanks - then we're talking about the same :) I need to look into the fact that `systools:make_relup` returns the `error` atom when `warnings_as_errors` is set - if it is a bug or not. Haven't had the time to do so yet. What I meant [above ](https://github.com/erlware/relx/pull/559#discussion_r96185064) was that maybe you need to do a workaround in the meantime - i.e. do as your original solution (when you didn't give the `warnings_as_error` option to systools, but handled it all in relx), but then add the cleanup if the relup is erroneously written. Unless you're happy with the `error` atom until further notice, that is...